### ISSUE TITLE
Order revisions by date option is gone.

### ISSUE THREAD

**Issue by Thuffir** [gitextensions/gitextensions#6411]
## Current behaviour

Option `View -> Order revisions by date` is missing in 3.x.x

## Expected behaviour
In 2.x.x I was able to switch between topological and date ordering. Now with 3.x.x it is gone.

## Steps to reproduce

Switch from 2.x.x to 3.x.x

## Screenshots

In **2.51.05**:
![image](https://user-images.githubusercontent.com/6106671/55064078-a3bf5100-5079-11e9-8a56-8abcf9d25e0b.png)

In **3.0.2** it looks like this:
![image](https://user-images.githubusercontent.com/6106671/55064324-0f092300-507a-11e9-8cbf-ca4eaaefe456.png)

## Did this work in previous version of GitExtensions

Yes, it was working throughout the 2.xx series. 

## Environment

- Git Extensions 3.0.2.5232
- Build 23b6f51905006ccdde8701591df706284b4155dc
- Git 2.20.1.windows.1
- Microsoft Windows NT 10.0.17763.0
- .NET Framework 4.7.3324.0
- DPI 96dpi (no scaling)

## Remarks
#5594 seems to be related, but I cannot find any infos why this option is gone now.

**Comment by gerhardol** [comment]
duplicate of #6310 
Also reported in #6395

**Comment by RussKie** [comment]
I don't think it is related, this one is about the revision graph not branches.

**Comment by RussKie** [comment]
The functionality was removed in 12ee816c1322b4a062e50aa2ba2b0356fbafe724, unfortunately @spdr870 didn't provide a good commit message to elaborate on the reasons.

My understanding this option no longer makes sense since the graph is rendered in the chronological order.

**Comment by Thuffir** [comment]
> Yes, this was intentionally removed to speedup graphloading

I think users should have the ability to toggle this option if they need it. (Even only temporarily) It is a very useful view for cherry picking and branch comparision imho.

**Comment by Thuffir** [comment]
Does it make sense to create a feature request out of this?

**Comment by RussKie** [comment]
Revisions are already sorted by date.

**Comment by Thuffir** [comment]
> Revisions are already sorted by date.
Thats true, but one cannot order them by topology like before. And that is what is missing for me.
